opflexagent/config.py

```diff
@@ -57,6 +57,21 @@
     cfg.StrOpt('fabric_bridge', default='br-fabric',
                help=_("The name of the bridge which connects to the ACI "
                       "fabric")),
+    cfg.StrOpt('bridge_manager',
+               default=('opflexagent.utils.bridge_managers.'
+                        'ovs_manager.OvsManager'),
+               help=_("The class to use for bridge management. "
+                      "Options are:"
+                      "opflexagent.utils.bridge_managers."
+                      "ovs_manager.OvsManager' (default), "
+                      "opflexagent.utils.bridge_managers."
+                      "vpp_manager.VPPManager")),
+]
```

Comment:
can we use entry points instead of full paths? This would require adding them to the setup.cfg file.

Reply:
Sure.. will do..